ambari-web/app/styles/bootstrap_overrides.less

```diff
@@ -439,6 +439,8 @@ select.form-control {
 .navigation-bar-container.collapsed ul.nav.side-nav-footer li ul.sub-menu {
   padding: 5px 0;
   background-color: #323544;
+  max-height: calc(100% - 100px);
```

Comment:
Incorrect format of rule, should be:
max-height: ~"calc(100% - 100px)";

Reply:
Done